sda/cmd/finalize/finalize.md

```diff
@@ -93,33 +92,33 @@ if `*_TYPE` is `S3` then the following variables are available:
 If a backup location is configured it will perform backup of a file.
-When running, finalize reads messages from the configured RabbitMQ queue (default "accessionIDs").
+When running, `finalize` reads messages from the configured RabbitMQ queue (commonly: `accessionIDs`).
 For each message, these steps are taken (if not otherwise noted, errors halt progress and the service moves on to the next message):
 
-1. The message is validated as valid JSON that matches the "ingestion-accession" schema. If the message can’t be validated it is discarded with an error message in the logs.
-2. If the service is configured to perform backups i.e. the `ARCHIVE_` and `BACKUP_` storage backends is set. Archived files will be copied to the backup location.
+1. The message is validated as valid JSON that matches the `ingestion-accession` schema. If the message can’t be validated it is discarded with an error message in the logs.
+2. If the service is configured to perform backups i.e. the `ARCHIVE_` and `BACKUP_` storage backend are set. Archived files will be copied to the backup location.
    1. The file size on disk is requested from the storage system.
    2. The database file size is compared against the disk file size.
-   3. A file reader is created for the archive storage file, and a file writer is created for the backup storage file..
+   3. A file reader is created for the archive storage file, and a file writer is created for the backup storage file.
 3. The file data is copied from the archive file reader to the backup file writer.
 4. If the type of the `DecryptedChecksums` field in the message is `sha256`, the value is stored.
-5. A new RabbitMQ "complete" message is created and validated against the "ingestion-completion" schema. If the validation fails, an error message is written to the logs.
-6. The file accession ID in the message is marked as "ready" in the database. On error the service sleeps for up to 5 minutes to allow for database recovery, after 5 minutes the message is Nacked, re-queued and an error message is written to the logs.
+5. A new RabbitMQ `complete` message is created and validated against the `ingestion-completion` schema. If the validation fails, an error message is written to the logs.
+6. The file accession ID in the message is marked as *ready* in the database. On error the service sleeps for up to 5 minutes to allow for database recovery, after 5 minutes the message is Nacked, re-queued and an error message is written to the logs.
 7. The complete message is sent to RabbitMQ. On error, a message is written to the logs.
 8. The original RabbitMQ message is Ack'ed.
 
 ## Communication
 
-- Finalize reads messages from one RabbitMQ queue (default `accessionIDs`).
-- Finalize writes messages to one RabbitMQ queue (default `backup`).
-- Finalize assigns the accession ID to a file in the database using the `SetAccessionID` function.
+- `Finalize` reads messages from one RabbitMQ queue (commonly: `accessionIDs`).
```

Comment:
```suggestion
- `Finalize` reads messages from one RabbitMQ queue (commonly: `accession`).
```

Reply:
this is where I am bit confused because helm charts use `accessionIDs` queue